rag_experiment_accelerator/doc_loader/tests/test_pdf_loader.py

```diff
@@ -8,15 +8,15 @@
 @pytest.mark.parametrize(
     "pypdf_enabled, expected_content",
     [
-        (True, "this is a sample pdf with text."),
+        (True, "col1  col2  col3  a b c 1 2 3        this is a sample text."),
         (
```

Comment:
Severity: Medium - The updated expected content string is more detailed, but the extra spaces can lead to confusion and potential test failures. Consider normalizing the whitespace or using a more predictable format to ensure consistency.


[Hack - peer-review-pal] We are sorry for the intrusion. Feel free to ignore/delete the comments with the tag - they are automatically generated by [our Hack project](https://github.com/microsoft/peer-review-pal).
If you find them useful, want us to delete them or have any feedback or know how to make it better and want to contribute - reach out to some of us: juliame@microsoft.com vadimkirilin@microsoft.com oteslenko@microsoft.com vimeena@microsoft.com
You can prevent future comments from appearing by adding #prp-ignore to your PR description.